### PR TITLE
feat: event ingestion for multiple plans

### PR DESCRIPTION
**Pull request by lovrocolic**

### CODE REVIEW

File: app/services/events_service.rb [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 class EventsService < BaseService
   def validate_params(params:)
-    mandatory_augments = %i[transaction_id customer_id code]
+    mandatory_augments = %i[transaction_id code]
```
Comment (rsempe) [inline app/services/events_service.rb:5]: Is that a typo to `mandatory_arguments`? 😅

File: app/services/events_service.rb [hunk 2]
```diff
@@ -2,7 +2,18 @@
...
+
+    missing_arguments = mandatory_augments.select { |arg| params[arg].blank? }
+    missing_subscription_arguments = mandatory_subscription_arguments.select { |arg| params[arg].blank? }
+    return result if missing_arguments.blank? && missing_subscription_arguments.count <= 1
+
+    result.fail!('missing_mandatory_param', nil, missing_arguments + missing_subscription_arguments)
+  end
+
+  def validate_batch_params(params:)
+    mandatory_augments = %i[transaction_id code subscription_ids]
```
Comment (rsempe) [inline app/services/events_service.rb:16]: Same here

File: app/services/events_service.rb (hunk 1 above)
Comment (vincent-pochet) [reply app/services/events_service.rb:5]: Probably mine :facepalm: :D

**Review by vincent-pochet — COMMENTED** [review]
Small comments, otherwise, great job! :rocket:

File: app/serializers/v1/event_serializer.rb [hunk 3]
```diff
@@ -10,6 +10,7 @@ def serialize
         code: model.code,
         timestamp: model.timestamp.iso8601,
         properties: model.properties,
+        subscription_id: model.subscription_id,
```
[the remark this answers is not in the document]
Comment (lovrocolic) [reply app/serializers/v1/event_serializer.rb:13]: Make sense, I will rename it. However I'm not sure about unique_id, I exposed it already in subscription serilizer...

**Comment by lovrocolic** [comment]
> Small comments, otherwise, great job! 🚀

Thanks, just added improvements based on comments!

**Review by vincent-pochet — APPROVED** [review]

**Pull request merged**